edx_rest_framework_extensions/utils.py

```diff
@@ -1,12 +1,26 @@
 """ Utility functions. """
 import logging
-
 import jwt
+
+from django.conf import settings
+
 from rest_framework_jwt.settings import api_settings
 
 logger = logging.getLogger(__name__)
 
 
+def get_jwt_issuers():
+    """
+    Retrieves the JWT issuers from settings
+    """
+    return settings.JWT_AUTH['JWT_ISSUERS'] or [
+        {
+            'JWT_ISSUER': api_settings.JWT_ISSUER,
```

Comment:
The default case, `JWT_ISSUERS` not set, needs to be tested. These key names are incorrect.